README.md

```diff
@@ -140,9 +140,6 @@ the `np` module returned will raise an error on attribute access. Using
 this feature is not all-or-nothing: One module may rely on one version of
 numpy, while another module may not set any requirement.
 
-_Note that the requirement must use the package [distribution name][] instead
-of the module [import name][]. For example, the `pyyaml` distribution provides
+_Note that the requirement must use the package [distribution name](https://www.sex.com/pin/57183088-teaching-her-to-behave/) instead
```

Comment:
Why is this adding a link to a sex site?

Reply:
I ran this locally on my computer and didn't get any porn spam!

Reply:
@Borda I am closing this PR.

It looks like you are a regular open source contributor. I worry your computer maybe hacked or something. Please check your computer and make sure you haven't inadvertently made any other PRs with porn links to other repos.

Reply:
I am happy to keep the discussion going and if there is a bug in mdformat, it would be great to get it fixed.

Reply:
Good catch, thank you.
 I allowed ci to run after checking the first commit, but then allowed it again after the second one without looking into the diff. Lesson has been learnt.

Reply:
My apology, no idea what happened, the section suppose to be:
```
_Note that the requirement must use the package [distribution name](https://packaging.python.org/en/latest/glossary/#term-Distribution-Package) instead
of the module [import name](https://packaging.python.org/en/latest/glossary/#term-Import-Package). For example, the `pyyaml` distribution provides
the `yaml` module for import._
```

Reply:
I couldn't reopen it since there was a force-push after it was closed. But I could open a new PR from your branch. I have a few things that I will mainly change if we decide to merge this. I will take a closer look at the changes because if we merge this, I will probably do the same thing in a bunch of repositories. Thanks!!

Reply:
Yes, it is handy hook; it also supports Black formatting, see:
https://github.com/Lightning-AI/torchmetrics/blob/d6b5b9883928df6d9bdfddf60b339ea24ba71ff7/.pre-commit-config.yaml#L72-L75